db/db_bloom_filter_test.cc

```diff
@@ -27,7 +28,17 @@
 namespace ROCKSDB_NAMESPACE {
 
 namespace {
-using BFP = BloomFilterPolicy;
+std::shared_ptr<const FilterPolicy> Create(double bits_per_key,
+                                           const std::string& name) {
+  return BloomLikeFilterPolicy::Create(name, bits_per_key);
+}
```

Comment:
Is there a reason for this method rather than just calling the BloomLikeFilterPolicy::Create directly?

Reply:
I didn't want to go through and manually re-order all the parameters. And on net it saves 6 lines of code (10 wraps saved for 4 for the definition).